apis/apps/v1beta3/emqxbroker_webhook.go

```diff
@@ -126,9 +126,15 @@ func (r *EmqxBroker) ValidateDelete() error {
 
 func validateTag(image string) error {
 	str := strings.Split(image, ":")
-	if len(str) > 1 {
-		match, _ := regexp.MatchString("^[0-4]+.[0-3]+.*$", str[1])
-		if match {
+	l := len(str)
+	if l > 1 {
+		match, _ := regexp.MatchString("^[0-4]+.[0-3]+.*$", str[l-1])
+		if !match {
+			match, _ = regexp.MatchString("^latest$", str[l-1])
+			if match {
+				return nil
```

Comment:
Why we need check "latest" ?  I don't understand this code, it's seems if match, then return "nil", if not match, still return "nil"